public/pages/DetectorsList/Containers/ConfirmActionModals/ConfirmActionModals.tsx

```diff
@@ -0,0 +1,405 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import {
+  EuiOverlayMask,
+  EuiCallOut,
+  EuiLink,
+  EuiIcon,
+  EuiText,
+  EuiFieldText,
+  EuiButton,
+  EuiButtonEmpty,
+  EuiModal,
+  EuiModalHeader,
+  EuiModalFooter,
+  EuiModalBody,
+  EuiModalHeaderTitle,
+  EuiDataGrid,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+// @ts-ignore
+import { toastNotifications } from 'ui/notify';
+//@ts-ignore
+import chrome from 'ui/chrome';
+import { getAlertingMonitorListLink } from '../../../../utils/utils';
+import { Monitor } from '../../../../models/interfaces';
+import { DetectorListItem } from '../../../../models/interfaces';
+import { PLUGIN_NAME } from '../../../../utils/constants';
+import { Listener } from '../../../../utils/utils';
+import { get } from 'lodash';
+import { containsEnabledDetectors } from '../../utils/helpers';
+import { EuiSpacer } from '@elastic/eui';
+
+interface ConfirmStartDetectorsModalProps {
+  detectors: DetectorListItem[];
+  hideModal(): void;
+  onStartDetectors(): void;
+  isListLoading: boolean;
+}
+
+interface ConfirmStopDetectorsModalProps {
```

Comment:
Can we split these three confirm modals into different files and put shared functions like `getNamesAndMonitorsGrid` into utils to make code more readable?

Reply:
sure, I will break up and add a `utils/helpers` file